README.md

```diff
@@ -1,82 +1,298 @@
+cd referee_hub
+```
+
+#### 2. Build the Frontend
+
+```bash
+cd src/frontend
+
+# Install dependencies
+yarn install --immutable
+
+# Build for development
+yarn build:dev
+
+# Or build for production
+yarn build:prod
+```
+
+The frontend build includes:
+- JavaScript bundling with Webpack
+- CSS compilation with PostCSS and Tailwind
+- Image asset copying
+
+#### 3. Build the Backend
+
+```bash
+cd src/backend
+
+# Restore dependencies
+dotnet restore
+
+# Build the solution
+dotnet build
+```
+
+The backend build automatically includes the frontend assets if they're available in `src/frontend/dist/`.
+
+#### 4. Run the Application
+
+**Development Mode (In-Memory Dependencies):**
+
+```bash
+cd src/backend/ManagementHub.Service
+
+# Run with hot reload
+dotnet run
+```
+
+The service will start on `http://localhost:5000` with in-memory database and cache.
+
+**With Docker Compose (Real Dependencies):**
+
+For a more production-like environment with PostgreSQL, Redis, and MailHog:
+
+```bash
+cd docker/staging
+docker compose up -d
+```
+
+The service will be available at `http://localhost:80`.
+
+### Running Tests
+
+**Frontend Tests:**
+
+```bash
+cd src/frontend
+
+# Run tests with coverage
+yarn test
+
+# Run tests in CI mode
+yarn test:ci
+```
+
+**Backend Tests:**
+
+```bash
+cd src/backend
+
+# Run all unit tests
+dotnet test
+
+# Run with verbose output
+dotnet test --verbosity normal
+```
+
+### Linting
+
+**Frontend:**
+
+```bash
+cd src/frontend
+
+# Run ESLint
+yarn lint
+
+# Run ESLint with CI reporting
+yarn lint:ci
+```
+
+### Building Docker Image
+
+To build a production Docker image:
+
+```bash
+cd src/backend
+
+# Publish and build Docker image
+dotnet publish --os linux --arch x64 -c Release -p:PublishProfile=DefaultContainer
+```
+
+The image will be tagged as `iqasport/management-hub:latest`.
+
+### API Documentation
+
+When running the application in Development mode, Swagger UI is available at:
+- `http://localhost:5000/swagger`
+
+The API documentation is automatically generated from the C# code using Swashbuckle.
+
+### Additional Documentation
+
+- **Building Details**: See [docs/building.md](docs/building.md) for detailed build instructions
+- **Testing Guide**: See [docs/testing.md](docs/testing.md) for testing information
+- **API Client**: See [docs/api-client.md](docs/api-client.md) for API client generation
+- **AWS Deployment**: See [docs/aws.md](docs/aws.md) for AWS-specific deployment details
+
+## Open Questions
+
+The following questions should be addressed during PR review:
+
+1. **Database Migrations**: Should we document the database migration process for local development?
+2. **Environment Variables**: Should we provide a template `.env` file with required environment variables?
+3. **SSL/HTTPS**: Should we document local HTTPS setup for testing Stripe integrations?
+4. **Seed Data**: Should we provide instructions for seeding test data for local development?
```

Comment:
The data is auto seeded in dev, let's only include info on test users (all have password "password") which you can find in the seed database csharp file